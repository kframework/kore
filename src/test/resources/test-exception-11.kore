[]

module TEST-EXCEPTION-11

  axiom{}
    #X:#S{} []

endmodule []
