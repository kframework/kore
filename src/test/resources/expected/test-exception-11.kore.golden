Parse error: "../../../test/resources//test-exception-11.kore" (line 6, column 21):
unexpected "["
expecting white space
metaSortConverter: Non empty parameter sorts..